pkg/justification/processor_test.go

```diff
@@ -133,10 +134,19 @@ func TestCreateToken(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	jvsKeyConfig := config.JVSKeyConfig{KeyName: key}
+	configBytes, err := json.Marshal(jvsKeyConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+	keyConfig, err := testutil.NewFakeRemoteConfig(string(configBytes), "processor_test_mock_remote_config.json")
```

Comment:
Do we need to clean up the file after test?

Reply:
I don't think we need to clean it since we are using `MemMapFs`(fyi: https://pkg.go.dev/github.com/spf13/afero#section-readme) it said `No test cleanup needed`